paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutInteractor.kt

```diff
@@ -86,6 +102,15 @@ internal class DefaultPaymentMethodVerticalLayoutInteractor(
         return supportedPaymentMethods.indexOfFirst { it.code == selectedPaymentMethodCode }
     }
 
+    private fun getDisplayedSavedPaymentMethod(
+        paymentMethods: List<PaymentMethod>?,
+        paymentMethodMetadata: PaymentMethodMetadata,
+        mostRecentlySelectedSavedPaymentMethod: PaymentMethod?,
+    ): DisplayableSavedPaymentMethod? {
+        val paymentMethodToDisplay = mostRecentlySelectedSavedPaymentMethod ?: paymentMethods?.firstOrNull()
```

Comment:
I thought we stored something (on disk), for the last used payment method. I think this is where we should read that.